src/main/java/io/github/aquerr/rentacar/domain/reservation/ReservationService.java

```diff
@@ -22,4 +27,9 @@ public Reservation createNewReservation(Reservation reservation) {
     public Reservation getReservation(Long reservationId) {
         return reservationConverter.toReservationDto(this.reservationRepository.findById(reservationId).orElse(null));
     }
+
+    public List<Reservation> getMyselfReservation() {
```

Comment:
Jeśli nie chcemy przekazywać Id usera do RESTa to czy moglibyśmy zmienić nazwę na „getMyReservations”?